src/utilities/graphql/transform.ts

```diff
@@ -10,9 +10,20 @@ const TYPENAME_FIELD: FieldNode = {
 };
 
 /**
- * Adds `__typename` to all selection sets in the document.
+ * Adds `__typename` to all selection sets in the document. The operation
```

Comment:
```suggestion
 * Adds `__typename` to all selection sets in the document. Beyond that, the operation
```

The original sentence confused me a bit - is this what you mean?